api/jest-dynalite-config.js

```diff
@@ -168,6 +168,47 @@ module.exports = {
         WriteCapacityUnits: 1,
       },
     },
+    {
```

Comment:
Just noting that if you do add user ID as a GSI, it would be good to include here.